firmware/crypto.c

```diff
@@ -110,6 +110,12 @@ int gpgMessageSign(HDNode *node, const uint8_t *message, size_t message_len, uin
 	}
 }
 
+int ecdsaMessageSign(HDNode *node, const uint8_t *message, size_t message_len, uint8_t *signature)
```

Comment:
This is the same as `sshMessageSign`, but I used this name for clarity. Let me know if you'd prefer I remove `sshMessageSign` and replaced it with this function.